### PR TITLE
write README.md

### PR DESCRIPTION
**Pull request by hkdnet**
write some rules...
- [x] wip workflow
- [x] devenv
- [x] code review
- [x] emoji

### CODE REVIEW

**Comment by hkdnet** [comment]
please review:star:

File: README.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
 # 3chLine
 
 a [Sails](http://sailsjs.org) application
```
Comment (keicy) [inline README.md]: please remove　this line.

Comment (hkdnet) [inline README.md]: we talk about our app name and rename `3chLine` -> `3chPiazza`
so i'll remove l.3 and modify app name :+1:

**Comment by hkdnet** [comment]
@keicy please review :hand:

File: README.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+* sails/node.js application
+
+## rules
+
+### how to develop
+
+follow this guideline
+[初心者向けWIPワークフロー&レビュー時の注意](http://qiita.com/sue738/items/09d9292e7654747ca208)
+
+what is the most important is _code review_
```
Comment (keicy) [inline README.md]: is it a little better of "The most important thing is ~" ?

Comment (hkdnet) [inline README.md]: it's simple.
ok i'll modify it :runner:

File: README.md [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+
+## rules
+
+### how to develop
+
+follow this guideline
+[初心者向けWIPワークフロー&レビュー時の注意](http://qiita.com/sue738/items/09d9292e7654747ca208)
+
+what is the most important is _code review_  
+we review our code and make our app cool :smiley:
```
Comment (keicy) [inline README.md]: "We review our code each other and ~" is a litter better?

**Comment by hkdnet** [comment]
Modify a phrase and the first characters each line are replaced with upper case.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,19 @@
-# 3chLine
+# 3chPiazza
 
-a [Sails](http://sailsjs.org) application
+* tw*tter clone for 3ch
```
Comment (keicy) [inline README.md]: "Communication "Piazza" for 3ch."  is just describing this product's feature, don't you?
i wanna remove other product's name :)

**Comment by hkdnet** [comment]
thanks your advise, @keicy !
please review :+1:

**Comment by keicy** [comment]
thanks for your good job! :+1: 
i do merge it.

**Comment by hkdnet** [comment]
sorry, this PR did close but was not merged :crying_cat_face:  
i can't figure out why, so i reopen and merge...
please revert if you don't like this @keicy :pray:

**Pull request merged**